README.md

```diff
@@ -0,0 +1,18 @@
+This repository contains [a list](config.json) of audit findings to be
+suppressed/ignored across [brave-core](https://github.com/brave/brave-core)
+branches.
+
+The identifier of a finding is a url, e.g.:
```

Comment:
Hmm, not sure this is correct, actually, I see we previously used just the RUSTSEC-... part